diff --git a/rpm/postgresql-common.spec b/rpm/postgresql-common.spec
index 96d8567..6b197c0 100644
--- a/rpm/postgresql-common.spec
+++ b/rpm/postgresql-common.spec
@@ -1,3 +1,5 @@
+%define _unpackaged_files_terminate_build 0
+
 Name:           postgresql-common
 Version:        %{version}
 Release:        1%{?dist}
@@ -96,7 +98,7 @@ cp debian/postgresql-common.logrotate %{buildroot}/etc/logrotate.d/postgresql-co
 %config /etc/postgresql-common/createcluster.conf
 %config /etc/logrotate.d/postgresql-common
 
-%if 0%{?rhel} >= 7
+%if 0%{?rhel} >= 7 || 0%{?fedora}
 %config /lib/systemd/system/*.service
 %config /lib/systemd/system/*.timer
 %config /lib/systemd/system-generators/postgresql-generator

